user-service/src/main/resources/db/migrations/V1__Create_user_table.sql

```diff
@@ -6,4 +6,11 @@ CREATE TABLE IF NOT EXISTS users (
     first_name VARCHAR(50),
     last_name VARCHAR(50),
     created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS user_favorite_movies (
```

Comment:
avoid modifying existing migrations to prevent issues in applied environments and loss of change history
add a new migration for creating the user_favorite_movies